doc/getting-started.md

```diff
@@ -9,133 +9,142 @@ mike@indexdata.com
 * [To run](#to-run)
 * [To initialize](#to-initialize)
 * [To use](#to-use)
-* [Integrating into a FOLIO-backend Vagrant box](#integrating-into-a-folio-backend-vagrant-box)
-* [To unassociate, undeploy and remove a running module](#to-unassociate-undeploy-and-remove-a-running-module)
-* [Adding permissions on the UI side](#adding-permissions-on-the-ui-side)
 
 ## Prerequisites
 
 You need
 [yaz4j](https://github.com/indexdata/yaz4j)
 and
 [YAZ](https://www.indexdata.com/yaz)
-before you can compile and install mod-copycat. In the directory next to your mod-copycat checkout, install as follows:
+before you can compile and install mod-copycat.
+
+Installing YAZ is easy on Ubuntu:
+
+	term1$ sudo apt install libyaz5
+
+yaz4j is distributed in https://maven.indexdata.com APT repo.
+If you are using Ubuntu, it is not necessary to compile and install yaz4j.
+
+Steps for install yaz4j (not necessary on Ubuntu amd64).
+In the directory next to mod-copycat checkout, install as follows:
 
 	term1$ sudo apt install yaz libyaz-dev swig
 	term1$ git clone https://github.com/indexdata/yaz4j.git
 	term1$ cd yaz4j
 	term1$ git checkout v1.6.0
```

Comment:
Please remind me why we invite people to lock to v1.6.0?

Reply:
Latest release. and it's easier to with with a fixed version.

Reply:
Eaiser to ... I guess, write the `pom.xml` or something.